resources/META-INF/plugin.xml

```diff
@@ -641,6 +641,7 @@
 
     <toolWindow id="Flutter Deep Links" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.deeplinks.DeepLinksViewFactory" />
     <toolWindow id="Flutter DevTools" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.RemainingDevToolsViewFactory" />
+    <toolWindow id="Flutter Extensions" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.DevToolsExtensionsViewFactory" />
```

Comment:
Whoops I missed this string. This should be "Flutter DevTools Extensions"

Reply:
oops fixed in new PR